src/Form/Form.js

```diff
@@ -88,14 +88,18 @@ class Form extends Util.mixin(BindMixin) {
       nextState.model = mergeNewModel(state.model, currentModel, nextModel);
     }
 
-    let currentErrors = this.buildStateObj(props.definition, 'showError');
-    let nextErrors = this.buildStateObj(nextProps.definition, 'showError');
-    if (!Util.isEqual(currentErrors, nextErrors)) {
-      nextState.erroredFields = mergeNewModel(
-        state.erroredFields,
-        currentErrors,
-        nextErrors
-      );
+    if (this.props.useExternalErrors) {
```

Comment:
yeah we do need this prop. for example, let's say you pass a definition to a form with no errors set. then through user interaction and the built-in form validation, it creates some errors. if we always pull errors from definition, the next time we render, it'll blow away that internal error.

a solution to this would be to redo the form so that it's stateless and just passes events back up to the parent and then we can fully just pull errors from the definition. i like that idea, but it's going to be too big of a change for now